CONTRIBUTING.md

```diff
@@ -508,6 +508,47 @@ In case you need to switch to sessionType=cookie, you
 - Use `AUTHGEAR_WEB_SDK_SESSION_TYPE=cookie` in your .env
 - Access the portal at port 8001 or 8011
 
+## Switch to Database config source
+
+1. In your `.env` set these values:
+
+```
+CONFIG_SOURCE_TYPE=database
+CUSTOM_RESOURCE_DIRECTORY=./hack/custom-resources
```

Comment:
I suggest we remove the directory `./hack`. It is a legacy directory that was used to host a local k8s API server.

Reply:
Looks like this file is still needed, or the new created projects will missing some required secrets

Reply:
Updated to use `./var` in the guide.